src/AbstractContainerCommandLoader.php

```diff
@@ -26,12 +27,15 @@ abstract class AbstractContainerCommandLoader implements CommandLoaderInterface
     /** @var ContainerInterface */
     private $container;
 
-    /** @var string[] */
+    /** @psalm-var array<string, string> */
     private $commandMap;
 
     final public function __construct(ContainerInterface $container, array $commandMap)
     {
-        $this->container  = $container;
+        $this->container = $container;
+
+        Assert::isMap($commandMap);
+        Assert::allString($commandMap);
```

Comment:
Is `$commandMap` allowed to be empty?

Reply:
Yes - absence of commands will still allow the application to run; you'll only be able to run "help" and "list", the built-in commands.